### PR TITLE
Api contract server capabilities

### PR DESCRIPTION
**Pull request by gokulakrishnansvm**
Adding API Contract Test Case for server_capabilities endpoint.
<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops Test

## What is the best way to verify this PR?

Step 1 : Follow steps from README file(traffic_ops/testing/api_contract) to run api-contract-tests.

Step 2 : To execute server_capabilities API Contract Testcase , navigate to traffic_ops/testing/api_contract/v4
pytest --to-user Username --to-password Password --to-url URL --config Config Path --request-template Request template --response-template Response template test_server_capabilities.py

Note: Config, Request template and response template paths are optional. 

## PR submission checklist
- [x] This PR has tests 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

**Review by TerekhovaKate — COMMENTED** [review]
please look at GitHub Actions 
`undefined label: tag-name`

**Comment by ocket8888** [comment]
> please look at GitHub Actions `undefined label: tag-name`

This is an extant bug unrelated to this PR. Needs to be fixed on master, but I just haven't gotten around to it yet.

**Review by ericholguin — CHANGES_REQUESTED** [review]

File: traffic_control/clients/python/trafficops/tosession.py [hunk 1]
```diff
@@ -1348,6 +1348,50 @@ def delete_parameter(self, parameter_id=None):
...
+		Create Server_capabilities
+		:ref:`to-api-server_capabilities`
+		:param data: The server_capabilities(s) data to use for server_capabilities creation.
+		:type data: Union[Dict[str, Any], List[Dict[str, Any]]]
+		:rtype: Tuple[Dict[str, Any], requests.Response]
+		:raises: Union[LoginError, OperationError]
+		"""
+
+
+	@api_request('put', 'server_capabilities', ('3.0', '4.0', '4.1', '5.0'))
```
[the remark this answers is not in the document]
Comment (gokulakrishnansvm) [reply traffic_control/clients/python/trafficops/tosession.py:1377]: Removed it.

**Review by ericholguin — APPROVED** [review]
All my comments have been addressed. Looks good.

**Pull request merged**